back/db/migrate/20230814115846_add_accuracy_and_input_ids_to_summaries.analysis.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# This migration comes from analysis (originally 20230814114510)
+
+class AddAccuracyAndInputIdsToSummaries < ActiveRecord::Migration[7.0]
+  def change
+    add_column :analysis_summaries, :accuracy, :float
+    add_column :analysis_summaries, :inputs_ids, :jsonb
```

Comment:
I'm using an array instead of a join table because:
- No referential integrity needed. If an input gets destroyed, we still want to know that something no longer existing was used for the summary.
- This is more 'logging' kind of data than data that's really used only local to the model or for later analysis
- can be large, join table would slow down creation without any benefits.